### ISSUE TITLE
Threading is broken for windows

### ISSUE THREAD

**Issue by twangboy** [saltstack/salt#21773]
2015.2 doesn't launch in windows. It looks like threading is broken.

This is the error I'm getting:

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\Python27\lib\multiprocessing\forking.py", line 380, in main
    prepare(preparation_data)
  File "C:\Python27\lib\multiprocessing\forking.py", line 489, in prepare
    file, path_name, etc = imp.find_module(main_name, dirs)
ImportError: No module named salt-minion
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\Python27\lib\multiprocessing\forking.py", line 380, in main
    prepare(preparation_data)
  File "C:\Python27\lib\multiprocessing\forking.py", line 489, in prepare
    file, path_name, etc = imp.find_module(main_name, dirs)
ImportError: No module named salt-minion
```

**Comment by rallytime** [comment]
@twangboy Was this fixed by #21777?

**Comment by twangboy** [comment]
It's fixed on 2015.2. We'll need to merge forward. I don't know if it's
broken on 2014.7.

On Wed, Mar 18, 2015 at 2:23 PM, Nicole Thomas <notifications@github.com>
wrote:

> @twangboy <https://github.com/twangboy> Was this fixed by #21777
> <https://github.com/saltstack/salt/pull/21777>?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/saltstack/salt/issues/21773#issuecomment-83155372>.
>



-- 
*Shane Lee*
*Windows Platform Engineer*
Salt Stack, Inc.

Join us at SaltConf15 <http://www.saltconf.com/> - March 3-5, 2015 - Salt
Lake City

**Comment by rallytime** [comment]
I don't think we implemented the feature that broke this in 2014.7, so I am going to close this as per @thatch45.
